src/components/dropdownSearch/tests/DropdownSearch.spec.tsx

```diff
@@ -287,6 +291,34 @@ describe('DropdownSearch', () => {
         expect(dropdownSearch.find('.dropdown-toggle').prop('disabled')).toBe(true);
       });
 
```

Comment:
the following tests are not the best ones ever, although I had to write them to keep the coverage of 100%